### DISCUSSION TITLE
Enhancement suggestion: Metacell metadata review

### DISCUSSION THREAD

**Discussion by tomgome** [tanaylab/MCView#117] [Ideas]
I think it might be handy to add a feature of querying a metacell's metadata upon selection by clicking on it or typing the ID.

originally the idea emerged from wanting to observe a Metacell top _n_ genes (not only the top 2 which are displayed), but it can also embed many other things (number of cells constructing it, expression attributes etc..), 
and it can be located in the Metacells tab.

**Comment by aviezerl** [comment]
Good idea. We might want to add it to the metadata tab which is coming up soon.
